src/lib/helpers.js

```diff
@@ -100,3 +100,28 @@ export const parseRelayOptions = options => {
 export const removeNulls = object => {
   Object.keys(object).forEach(key => object[key] == null && delete object[key]) // eslint-disable-line eqeqeq, no-param-reassign, max-len
 }
+// Validates Relay cursor or page/size pagination
+export const validatePagingParams = args => {
+  const { page, size, first, last } = args
+  if ((page && !size) || (size && !page)) {
```

Comment:
This block could be simplified to 
```javascript
if (page && size) {
  if (first || last)) {
    throw new Error(...) 
  }
} else {
  throw new Error("Must specify both a page and size param.")
}
```

Reply:
I don't think thats quite right, since you dont need page _or_ size (if youre using cursors).

Reply:
Ah, I thought page + size was always required by this logic

Reply:
Nah, you basically can either use cursor args, _or_ page/size. Then this is just some simple validation for dev sanity, essentially. So asserting you're using page/size properly, but you can also not use them at all.